src/components/Constants/UseConstants.tsx

```diff
@@ -0,0 +1,7 @@
+import { useQuery } from '@apollo/client';
+import { LoadConstantsDocument } from './LoadConstants.generated';
+
+// Use this call to force loading from cache storage first
+export const UseApiConstants = useQuery(LoadConstantsDocument, {
```

Comment:
Hooks are usually named with a lowercase `use`. I'd suggest naming the file `src/hooks/useApiConstants.tsx` since it is a shared hook. If it was tied to a specific part of the app, maybe it would be better to live in the component's folder.
```suggestion
export const useApiConstants = useQuery(LoadConstantsDocument, {
```
It's best to type this
```suggestion
export const useApiConstants = useQuery<LoadConstantsQuery, LoadConstantsQueryVariables>(LoadConstantsDocument, {
```
If you look in `LoadConstants.generated.ts`, codegen (running `yarn gql`) created `useLoadConstantsQuery` for you which implements all this for you 😄
```suggestion
export const useApiConstants = useLoadConstantsQuery({
```

You can probably make the return value of this hook even more convenient by pulling out `data.constant` and returning it.